src/pages/index.js

```diff
@@ -163,14 +97,44 @@ const Home = ({ data }) => {
           </Col>
         </Row>
       </Container>
+      
+      <section className="py-5">
```

Comment:
It's required so we can redirect from another route to this exact section's position

```suggestion
      <section className="py-5" id="demos">
```